CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to LinuxBoot/Heads
+
+Thank you for your interest in contributing to LinuxBoot/Heads! We welcome contributions
+from everyone. By participating in this project, respecting these contribution guidelines
+will ease and fasten their review, fixing and merging.
+
+## Communication Channels
+
+For the most up-to-date information on our community channels, please visit:
+https://osresearch.net/community/
+
+## Best Practices for Issues and Contributions
+
+Before opening an issue or submitting a contribution, please:
+
+1. Search existing resources (GitHub issues, wiki, Matrix channel history).
+2. Review the documentation at https://osresearch.net.
+3. Choose the appropriate channel for your contribution or question.
+
+### When Starting a Discussion on Matrix
+
+- Clearly state your topic or question.
+- Provide context, including your environment (OS, hardware, etc.) if relevant.
+- Be patient and respectful while waiting for responses.
+- Structure your topic effectively with a clear title and organized content.
+- Use code blocks for command outputs or error messages.
+- Consider breaking complex topics into smaller, manageable questions.
+- Tag your topic with relevant keywords.
+
+### When Opening an Issue on GitHub
+- Search globally on linuxboot organization for related keywords first.
+  - EG: yubikey search in issues: https://github.com/search?q=org%3Alinuxboot+yubikey&type=issues
+- Choose a clear and descriptive title.
+- Clearly describe the problem or suggestion:
+  - Summarize the issue or feature request.
+  - Explain current behavior and expected behavior.
+  - For bugs, provide steps to reproduce.
+  - For feature requests, explain use case and benefits.
+- Mention your environment details.
+- Use appropriate labels.
```

Comment:
I think labels can be set only by people who have some permissions in the repository.